lib/networkPacks/PacksForClientBattle.h

```diff
@@ -112,12 +111,14 @@ struct DLL_LINKAGE BattleResultAccepted : public CPackForClient
 	BattleID battleID = BattleID::NONE;
 	BattleSideArray<HeroBattleResults> heroResult;
 	BattleSide winnerSide;
+	std::vector<BulkMoveArtifacts> artifacts;
```

Comment:
In which scenarios multiple `BulkMoveArtifacts` packs might be needed? Why single BulkMoveArtifacts pack can't be used?

Reply:
Duplication is necessary while there are two network packets BattleResultAccepted and BattleResultsApplied. On the server side, we transfer artifacts while both heroes exist. And on the client side, after the left-hand side. After finishing work with the necromancy window and the eagle eye, there will be only one packet left.